cloudify_cli/cli/helptexts.py

```diff
@@ -88,6 +88,11 @@
     "In deployment update, perform install workflow and then uninstall "
     "workflow. default: uninstall and then install"
 )
+PREVIEW = (
+    "In deployment update, perform a preview of an update and not an actual "
+    "update. The changes that should be done will be returned, but no change "
+    "will actually take effect"
```

Comment:
This doesn't need to state that it's 'in deployment update'. If we feel that we need to track that then we should call this PREVIEW_UPDATE or similar instead. The user will see this in the helptext for the deployment update, so stating it in the message itself is redundant.

Also, the message might be better formatted something like:
"Preview the deployment update, stating what changes will be made without actually applying any changes." (or something similar)